packages/amplication-client/src/style/dark-css-variables.css

```diff
@@ -50,3 +50,15 @@
   --diffViewerTitleBorderColor: var(--gray-full);
   --screenResolutionMessageColor: var(--gray-base);
 }
+.spin {
+  animation: spin 1s infinite linear;
+}
```

Comment:
we have a CircularProgress component which we should use instead of implementing a local style for it

Reply:
Okay I'll change this one